.gitignore

```diff
@@ -38,15 +38,14 @@ test-results
 samples/_static/js/*.js*
 samples/_static/js/*
 samples/*/public/app.*
-samples/*/todos
 samples/*/users
-todos/
-users/
+!samples/angular-todo/*/todos
```

Comment:
Huh, that's odd. I thought the below `todos/` would only ignore a directory created at the root. Now I'm wondering how [the `todos` directory in the React sample](https://github.com/ForgeRock/forgerock-javascript-sdk/tree/develop/samples/reactjs-todo/client/components/todos) survived.

Reply:
Me neither!

Reply:
For sanity let's add the same exception for the react app